src/components/layout/PageBtns.js

```diff
@@ -0,0 +1,100 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+
+const PageBtns = ({
+  totalItems,
+  hasMore,
+  next,
+  fetchData,
+  paginationURL,
+  pageSize,
+}) => {
+  // a variable to keep track of the previous page
+  const [prev, setPrev] = useState(null);
+
+  useEffect(() => {
+    setPrev(
+      next
+        ? next - pageSize * 2 - 1
+        : totalItems - (pageSize + (totalItems % pageSize))
+    );
+    //eslint-disable-next-line
+  }, [next]);
```

Comment:
I'm pretty sure you can add `pageSize` and `totalItems` to the dependency array here with no issues since they are basically constants.

Reply:
fixed in e968a49